website/common/script/content/faq.js

```diff
@@ -14,7 +14,11 @@ for (let i = 0; i <= NUMBER_OF_QUESTIONS; i++) {
   let question = {
     question: t(`faqQuestion${i}`),
     ios: t(`iosFaqAnswer${i}`),
-    web: t(`webFaqAnswer${i}`),
+    web: t(`webFaqAnswer${i}`, {
+      //  TODO: Need to pull these values from nconf
```

Comment:
Ah okay... with the current client side the only way to access env vars is in Jade templates but not here in a JS file. I'll be sure to add it to the rewrite of the client side that we're doing